common/src/main/java/com/wynntils/screens/base/widgets/ItemSearchHelperWidget.java

```diff
@@ -29,14 +29,13 @@ public ItemSearchHelperWidget(
     @Override
     public List<Component> getTooltipLines() {
         List<Component> helpTooltip =
-                new ArrayList<>(List.of(Component.translatable("feature.wynntils.containerSearch.tooltip")));
+                new ArrayList<>(List.of(Component.translatable("screens.wynntils.itemSearchHelperWidget.name")));
```

Comment:
Maybe this should become `screen.wynntils...` for consistency. Or we can do the refactor in some other pr.